### DISCUSSION TITLE
【语雀】文档历史版本对比，diff一目了然

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#79] [1️⃣ 新鲜事]
语雀的文档历史可以分为三种：

|  编辑时自动保存   | 发布时自动保存  | 手动保存并取个版本名 |
|  ---  | ---  | --- |
| <img width="518" alt="image" src="https://user-images.githubusercontent.com/32434520/172281717-5272cc2d-4ee9-4e0b-9399-be255bdfc97d.png">  | <img width="425" alt="image" src="https://user-images.githubusercontent.com/32434520/172281811-875c87e2-b020-4591-964b-143599f2a7fe.png"> | <img width="413" alt="image" src="https://user-images.githubusercontent.com/32434520/172281859-b28b4b4b-37b7-45bb-a88b-1e90f54c1d5e.png"> | <img width="1343" alt="image" src="https://user-images.githubusercontent.com/32434520/172281270-7f9003f7-c312-496f-bd1c-b69a9d252338.png"> |

文档历史除了可以方便的做版本回退外，还可以进行版本对比，就像git的diff或word的审阅一样。这样可以一目了然看出修改了哪些地方，如图：

<img width="1360" alt="image" src="https://user-images.githubusercontent.com/32434520/172282547-752a81ed-ec94-4945-9e6a-6d9917682093.png">

**Comment by will-ww** [comment]
很棒，特别符合开发者的心思与感觉~

**Comment by AliceCodeZhang** [comment]
这个确实优秀
